Makefile

```diff
@@ -224,7 +224,7 @@ clidocsgen:
 
 
 .PHONY: codegen-local
-codegen-local: mod-vendor-local gogen protogen clientgen openapigen clidocsgen manifests-local notification-docs notification-catalog
+codegen-local: mod-vendor-local gogen protogen clientgen clidocsgen manifests-local notification-docs notification-catalog
```

Comment:
I don't think this should change actually; `openapigen` should still run.

Reply:
@sivchari did you see my comment above? ☝️

Reply:
@blakepettersson 
Oh, I've completely overlooked.
I fixed it.

Reply:
@blakepettersson 

openapigen has already been deleted. So, if openapigen is remained, the CI has never been passed.

ref: https://github.com/argoproj/argo-cd/pull/17491#issuecomment-2043643951